networkx/algorithms/euler.py

```diff
@@ -214,10 +214,10 @@ def eulerian_circuit(G, source=None, keys=False):
 
 
 def has_eulerian_path(G, source=None):
-    """Return True iff `G` has an Eulerian path.
+    """Return True iff `G` has an Eulerian path
 
     An Eulerian path is a path in a graph which uses each edge of a graph
-    exactly once.
+    exactly once. The input `source` is the node at which the path starts.
```

Comment:
```suggestion
    exactly once. If `source` is specified, then this function checks
    whether an Eulerian path that starts at node `source` exists.
.```

Just a wording suggestion.